perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/ServicesManagerMethod.java

```diff
@@ -371,7 +371,8 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 			parms.stateChangingCheck();
 
 			ac.getServicesManager().deleteService(ac.getSession(),
-					ac.getServiceById(parms.readInt("service")));
+					ac.getServiceById(parms.readInt("service")),
+					parms.contains("force") ? parms.readBoolean("force") : false);
```

Comment:
Javadoc for this method should be changed too. You want to have 2 ways how to call this method - one with force parameter, one without.

Reply:
Added.